.ci/docker/ci_commit_pins/pytorch.txt

```diff
@@ -1 +1 @@
-295f2ed4d103017f7e19a7b8263ece606cd629db
+cdb42bd8cc05bef0ec9b682b274c2acb273f2d62
```

Comment:
where did you get this? 

https://hud.pytorch.org/pytorch/pytorch/commit/cdb42bd8cc05bef0ec9b682b274c2acb273f2d62

It should be the other pin 

7ae0ce6360b6e4f944906502d20da24c04debee5

Reply:
I copied the title `PyTorch commit: cdb42bd8cc05bef0ec9b682b274c2acb273f2d62`, wrong sha. Will update it